pkg/pod/status_test.go

```diff
@@ -1290,3 +1167,36 @@ func TestMarkStatusSuccess(t *testing.T) {
 		t.Errorf("Unexpected status: %s", diff.PrintWantGot(d))
 	}
 }
+
+func statusRunning(t *testing.T) duckv1beta1.Status {
+	t.Helper()
+	var trs v1beta1.TaskRunStatus
+	MarkStatusRunning(&trs, v1beta1.TaskRunReasonRunning.String(), "Not all Steps in the Task have finished executing")
+	return trs.Status
+}
+
+func statusFailure(t *testing.T, message string) duckv1beta1.Status {
+	t.Helper()
+	var trs v1beta1.TaskRunStatus
+	MarkStatusFailure(&trs, message)
+	return trs.Status
+}
+
+func statusSuccess(t *testing.T) duckv1beta1.Status {
+	t.Helper()
+	var trs v1beta1.TaskRunStatus
+	MarkStatusSuccess(&trs)
+	return trs.Status
+}
+
+func statusPending(t *testing.T, reason, message string) duckv1beta1.Status {
+	t.Helper()
+	return duckv1beta1.Status{
+		Conditions: []apis.Condition{{
+			Type:    apis.ConditionSucceeded,
+			Status:  corev1.ConditionUnknown,
+			Reason:  reason,
+			Message: message,
+		}},
+	}
+}
```

Comment:
All those methods do not need `t *testing.T` :angel:

Reply:
yes `t.Helper()` needs access to `*t testing.T` variable

Reply:
@pritidesai no need to call `t.Helper` if the function itself doesn't *use* `*testing.T`. `t.Helper()` is only useful (and needed) if you want the *report* (`t.Log`, `t.Error`, …) done in a function to be reported as the *caller* of this function. Here, we don't use `t *testing.T` at all in these functions, so we don't need to call `t.Helper()`, and so we don't need to pass `*testing.T`